src/style.css

```diff
@@ -546,4 +567,58 @@ button.waves-effect.waves-light.btn-small.btn-login {
     text-align: center;
     line-height: 20px;
     margin-bottom: 50px;
-}
\ No newline at end of file
+}
+
+/* *********MediasQuerys*********** */
+
+@media (min-width: 1280px) {
```

Comment:
Bien por el uso de medias queries pero el orden está invertido es mejor seguir un patron de diseño cómo mobile first: https://zellwk.com/blog/how-to-write-mobile-first-css/